### DISCUSSION TITLE
boxlambda/introducing-boxlambda/

### DISCUSSION THREAD

**Discussion by giscus** [epsilon537/boxlambda#2] [Announcements]
# boxlambda/introducing-boxlambda/

Alright, this is it. We’re live. I’m starting a project called BoxLambda. Here’s the run-down, copied verbatim from the README.md:

https://epsilon537.github.io/boxlambda/introducing-boxlambda/

**Comment by epsilon537** [comment]
Testing Giscus comment system.

**Reply by epsilon537** [reply]
Testing Giscus comment reply.

**Comment by qaziquza** [comment]
Very interesting project!
